### DISCUSSION TITLE
How to run a server side script without making a route?

### DISCUSSION THREAD

**Discussion by jervalles** [nuxt/nuxt#22591] [Questions]
I'm actually thinking about using Nuxt+Nitro (api) or Nuxt + Django and I'm listing the pros and cons.

I'm wondering, if I need to run a custom script from the backend, how can we do that without creating a route specialy for that?

For example, in django we can connect to the shell, even in production. 

If for example I would like to run a script once (like a database modification script)

thanks

**Comment by TheAlexLichter** [comment]
You could e.g. create a `scripts` folder with your integrations/scripts. They'd be decoupled from Nuxt/Nitro though.

**Reply by jervalles** [reply]
hi, thanks for your answer! but how can we run these scripts?

**Reply by TheAlexLichter** [reply]
Depends where you run your application, but e.g. via `node` (TS e.g. via jiti) ☺️

**Comment by angelhdzmultimedia** [comment]
You can create a `~/server/plugins/yourPlugin.ts`, a `~/server/index.ts`, or a `~/modules/yourModule.ts`.

I just created a `~/modules/typegoose.ts` module, to init my mongoose client and use @typegoose/typegoose class-based schemas/models.

![image](https://github.com/nuxt/nuxt/assets/10191085/45130798-f10a-4461-86a6-438598bb7643)
